NetCoreMentoring/src/NetCoreMentoring.App/Startup.cs

```diff
@@ -40,9 +49,14 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseHttpsRedirection();
             app.UseStaticFiles();
 
+            //app.UseMiddleware<ImageCacheMiddleware>();
```

Comment:
should be similar to:

        app.UseMiddleware<ImageCacheMiddleware>( 
                new CachingOptions 
                {
                    CachedCodePath = new System.Collections.Generic.Dictionary<string, string>
                    {
                        {  "/Category/GetPicture", "categoryId" }
                    }
                }
            );